scss/main.scss

```diff
@@ -1119,4 +1119,129 @@ p.cardText {
   width:100%;
   max-width: 395px;
   max-height: 261px;
-}
\ No newline at end of file
+}
+
+.quote-content-container {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 80px 96px;
+  max-width: 1440px;
+  height: max-content;
+  h1 {
+    padding: 16px 0px 16px 0px;
+    margin-bottom: 0px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 600px;
+    height: auto;
+    font-family: $font-family-base;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 62px;
+    color: #333333;
+  }
+  p {
+    margin-bottom: 24px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 600px;
+    height: auto;
+    font-family: $font-family-base;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 30px;
+    color: #333333;
+  }
+  @media (max-width: 768px) {
+    padding: 64px 48px;
+    h1 {
+      width: 672px;
+    }
+    p {
+      width: 672px;
+    }
+  }
+  @media (max-width: 651px) {
+    h1 {
+      font-size: 32px;
+      width: 555px;
+    }
+    p {
+      font-size: 18px;
+      line-height: 23px;
+      width: 555px;
```

Comment:
Between 360px and 651px, this paragraph is 555px, so it goes off the screen. Can we please fix?

Reply:
I don't see this issue on my end, perhaps it's resolved. Let me know if you're still seeing this.